tools/test_utils.py

```diff
@@ -348,7 +250,8 @@ def run_resnet50_infer_from_artifacts(artifact_dir, batch_size, iterations):
         install_ngraph_bridge(artifact_dir)
 
     # Check/download pretrained model
-    pretrained_models_dir = root_pwd + '/pretrained_models'
+    pretrained_models_dir = os.path.abspath(
```

Comment:
So you have the model downloaded once on each machine? In case some changes are made, we'll need to manually download or it'll happen automatically?

Reply:
The script downloads the pretrained model required for the test(s) if it doesn't exist already. This PR only fixes the location of the download.